images/win/scripts/Tests/Haskell.Tests.ps1

```diff
@@ -49,9 +49,8 @@ Describe "Haskell" {
         "cabal --version" | Should -ReturnZeroExitCode
     }
 
-    It "cabal config was modified and exists" {
-        $env:CABAL_DIR | Should -Exist
-        "cabal user-config diff" | Should -ReturnZeroExitCode
+    It "cabal folder does not exist" {
+        $env:CABAL_DIR | Should -Not -Exist

```

Comment:
I think we still set this env var... the main difference now is that `cabal user-config diff` should return an **empty string**.

Reply:
@hasufell , `Expected path 'C:\cabal' to exist, but it did not exist.` -  But this path doesn't exist after the changes.

Reply:
Ah right... but can we check whether the env var is set?

Reply:
Thank you. Done.